utils/nctl/sh/scenarios/configs/gov96.config.toml

```diff
@@ -10,6 +10,8 @@
 # If set, use this hash as a trust anchor when joining an existing network.
 #trusted_hash = 'HEX-FORMATTED BLOCK HASH'
 
+#archival_sync = false
```

Comment:
Why did you choose to comment this one out?

Reply:
Didn't @Fraser999 say I should comment this out [here][1]?

[1]: https://github.com/casper-network/casper-node/pull/1995#pullrequestreview-750135646